packages/tasit-action/src/contract/Action.js

```diff
@@ -6,33 +6,65 @@ import ConfigLoader from "../ConfigLoader";
 //  and/or MetaTxAction subclasses
 export class Action extends Subscription {
   #provider;
+  #signer;
+  #rawTx;
   #tx;
   #txConfirmations;
   #timeout;
   #lastConfirmationTime;
 
-  constructor(txPromise, provider) {
+  constructor(rawTx, provider, signer) {
```

Comment:
Creating a thread for that:

> > Maybe a better name could be used instead of rawTx (maybe rawAction?).

> Sounds good

To track easily.

Reply:
https://github.com/tasitlabs/TasitSDK/pull/360

Reply:
Sounds good, thanks